app/_data/docs_nav_gateway_3.0.x.yml

```diff
@@ -66,17 +66,17 @@ items:
       - text: Deployment Topologies
         items:
           - text: Overview
-            url: /production-deployment/deployment-topologies/
+            url: /production/deployment-topologies/
           - text: Hybrid Mode
             items:
             - text: Overview
-              url: /production-deployment/deployment-topologies/hybrid-mode/
+              url: /production/deployment-topologies/hybrid-mode/
             - text: Deploy Kong Gateway in Hybrid mode
-              url: /production-deployment/deployment-topologies/hybrid-mode/setup
+              url: /production/deployment-topologies/hybrid-mode/setup
           - text: DB-less Deployment
-            url: /production-deployment/deployment-topologies/db-less-and-declarative-config
+            url: /production/deployment-topologies/db-less-and-declarative-config
           - text: Traditional
-            url: /production-deployment/deployment-topologies/traditional
+            url: /production/deployment-topologies/traditional
       - text: Installation Options
```

Comment:
All of the pages in this section are missing "/production/" in the URL and "Production Deployment" in the breadcrumbs. I'm not sure if this is intentional or not (I'm guessing it's unintentional because these pages were moved here).

Reply:
This is intentional. Install was moved to the top level as installation instructions are technically used in dev as well as production, but kept in the Kong in Production section in the sidenav.

Expected URL is `/gateway/latest/install`